comid/rawvalue.go

```diff
@@ -27,6 +27,14 @@ func (o *RawValue) SetBytes(val []byte) *RawValue {
 	return o
 }
 
+func (o RawValue) GetBytes() ([]byte, error) {
+	if o.val == nil {
+		return nil, fmt.Errorf("raw value is not set")
+	}
+	t := o.val.(TaggedRawValueBytes)
```

Comment:
isn't this gonna explode if the underlying type doesn't match?

Reply:
ok, changing it now!